docs/modules/tests.md

```diff
@@ -0,0 +1,15 @@
+# Module: Tests
+
+All modules **MUST** have tests with minimum coverage.
+
+A newly created module **MUST** have tests either implemented using the following:
+
+- [Terraform test framework](https://developer.hashicorp.com/terraform/tutorials/configuration-language/test)
+- [Terratest](https://terratest.gruntwork.io/)
```

Comment:
We don't want to support this any more, right?

Reply:
You mean the terra tests? Our [tf-module-template](https://github.com/turo/tf-module-template/blob/main/test/main_test.go) repo is still using terratests 😐